### DISCUSSION TITLE
Score time display looks weird

### DISCUSSION THREAD

**Discussion by Morilli** [ppy/osu#20397] [Q&A]
When viewing a score in a leaderboard, the text that displays the score's age looks off. For example, "days" is abbreviated as "dys": 
![image](https://user-images.githubusercontent.com/35152647/191659475-c5ff06ec-3519-4ca7-af0a-406ac855d393.png)

It appears like there is more than enough space to display full words like "days", "months" or "years", and even if that isn't desired, the correct abbreviation for days should probably just be "d".

**Comment by peppy** [comment]
The abbreviation is standardised across all usages, some which don't have enough space for "days".

**Comment by respektive** [comment]
I think these abbreviations are a bit weird in general. Not sure if there is some specific reason for these abbreviations, but personally I would prefer using some, from my experience, more commonly used abbreviations:
| Unit | Abbreviation |
|---|---|
| Seconds | s |
| Minutes | min |
| Hours | h |
| Days | d |
| Weeks | wk |
| Months | mo |
| Years | y |
